deploy/README.md

```diff
@@ -0,0 +1,84 @@
+# Deployment Guide (Draft)
+
+## Prerequests
```

Comment:
I think you mean "Prerequisites".  "Prerequests" isn't a word

Reply:
😂I need a spell checking plugin